.travis.yml

```diff
@@ -15,6 +18,9 @@ jobs:
     - os: linux
       language: python
       python: 3.7
+      before_install: 
+              - sudo apt-get -y install ffmpeg
```

Comment:
I had to do something similar in #98. Out of curiosity, why is `-y` necessary here?

Reply:
Disables prompting the user for confirmation when installing the package.